modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -537,6 +537,29 @@ type CreateProgramResult {
   program: Program!
 }
 
+"""
+Input for creating a proposal.  Identify the program to update with one
+of `programId`, `proposalReference` or `programReference`.  If more than one of
+these is specified, all must match.  Use `SET` to specify the new program
+reference properties.
+"""
+input CreateProposalInput {
+  programId:         ProgramId
+  proposalReference: ProposalReferenceLabel
+  programReference:  ProgramReferenceLabel
```

Comment:
Seems like it would be odd to already have a proposal or program reference when creating a new proposal.

Reply:
Good point. I'll remove the references. I guess I got carried away by the boilerplate.

Reply:
Raises a question: what would happen if you create a proposal for a program that already has one?  Does it just reset it and become un-submitted / unaccepted if submitted / accepted?

Reply:
If you try to create a proposal for a program that already has one, you get an error. There is a test for that in `createProposal`